cv-19-i-need-help-test/EndToEndTests/V2/GetHelpRequestTest.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Collections.Generic;
+using Amazon.Lambda.APIGatewayEvents;
+using AutoFixture;
+using CV19INeedHelp.Boundary.V2;
+using CV19INeedHelp.Boundary.V2.Responses;
+using CV19INeedHelp.Data.V1;
+using CV19INeedHelp.Helpers.V2;
+using CV19INeedHelp.Models.V1;
+using CV19INeedHelpTest.TestHelpers;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace CV19INeedHelpTest.EndToEndTests.V2
+{
+    [TestFixture]
+    public class GetHelpRequestTest
+    {
+        private string _currentConnStr;
+        private Handler _handler;
+        private Fixture _fixture;
+        private Cv19SupportDbContext _dbContext;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _currentConnStr = Environment.GetEnvironmentVariable("CV_19_DB_CONNECTION");
+            const string connectionString = "Host=localhost;Database=i-need-help-test;Username=postgres;Password=mypassword";
+            _dbContext = new Cv19SupportDbContext(_currentConnStr ?? connectionString);
+            if (_currentConnStr == null) Environment.SetEnvironmentVariable("CV_19_DB_CONNECTION", connectionString);
+            _fixture = new Fixture();
+            CustomizeFixture.V2ResidentResponseParsable(_fixture);
+            _handler = new Handler();
+
+            AssertionOptions.AssertEquivalencyUsing(options =>
+            {
+                options.Using<DateTime>(ctx => ctx.Subject.Should().BeCloseTo(ctx.Expectation)).WhenTypeIs<DateTime>();
+                options.Using<DateTimeOffset>(ctx => ctx.Subject.Should().BeCloseTo(ctx.Expectation)).WhenTypeIs<DateTimeOffset>();
+                return options;
+            });
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            Environment.SetEnvironmentVariable("CV_19_DB_CONNECTION", _currentConnStr);
+            var addedEntities = _dbContext.ResidentSupportAnnex;
+            _dbContext.ResidentSupportAnnex.RemoveRange(addedEntities);
+            _dbContext.SaveChanges();
+        }
+
+        [Test]
+        public void ReturnsCamelCasedResponse()
+        {
+            var helpRequest = new ResidentSupportAnnex { Id = 1 };
+            _dbContext.ResidentSupportAnnex.Add(helpRequest);
+            _dbContext.SaveChanges();
+
+            var response = _handler.GetHelpRequest(new APIGatewayProxyRequest
+            {
+                PathParameters = new Dictionary<string, string> {{"id", "1"}}
+            }, null);
+
+            response.StatusCode.Should().Be(200);
+            response.Body.Should().BeEquivalentTo(@"{
+  ""id"": 1,
+  ""isDuplicate"": ""FALSE"",
+  ""recordStatus"": ""MASTER""
+}");
+        }
+
+        [Test]
+        public void WithNoQueryParameters_ReturnsNotFound()
```

Comment:
Maybe rename this to reflect the situation a bit better. I guess something like.. If the help request doesn't exist for given ID

Reply:
Changes applied in #34